code/src/Components/BirthMonth.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+
+const BirthMonth = ({ yourMonth, onMonthChange }) => {
+
+  return (
+    <div className="birthMonth">
+      <h3 tabIndex="0">BIRTH MONTH</h3>
+
+      <div className="radiobutton-section1">
+        <div className="radiobutton">
+          <label className="radiobutton-css" tabIndex="0" htmlFor="january">JAN</label>
+          <input
+            tabIndex="0"
+            id="Professional"
+            type="radio"
+            checked={yourMonth.includes('Professional')}
+            onChange={() => onMonthChange('Professional')}
+          />
+        </div>
```

Comment:
I got the message "Form Elements do not have associated labels" from the lighthouse report on the 
radiobuttons - I think the label is't connected to the input since the htmlFor and the Id has different values.  See link below for more info: 
https://web.dev/label/?utm_source=lighthouse&utm_medium=devtools